src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -89,16 +95,8 @@ IDisposable StartTimer()
             }
 
             var data = await service.ReadLocalData();
-            var changed = false;
-
-            if (firstTick)
-            {
-                var current = await GetCurrentReport(data);
-                current.Measures.NumberOfStartups++;
-                changed = true;
-                firstTick = false;
-            }
 
+            var changed = false;
```

Comment:
Because of this change... `WriteLocalData` will not be called for a startup...

Reply:
It will be called above in `IncrementCounter`